addons/mail/static/src/models/composer_view.js

```diff
@@ -182,6 +182,16 @@ registerModel({
             ev.preventDefault();
             this.discard();
         },
+        /**
+         * @private
+         * @param {MouseEvent} ev
+         */
+        onClickReplyingToMessage(ev) {
+            if (!this.threadView || !this.threadView.replyingToMessageView) {
+                return;
+            }
```

Comment:
Did you have the issue actually happen with either of those being undefined or is it just preventive?

While in theory they are justified, we typically don't guard click handlers unless the issue actually happens.

Reply:
Ya. It was just for preventive manner. Although, it is not creating any issues right now. So, I have removed it right now.